DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyBase.cs

```diff
@@ -209,7 +209,7 @@ public override void Initialize()
             catch (Exception ex)
             {
                 throw new Exception(string.Format("It seems the client configuration was not migrated to accommodate for the 'Tiberian Sun Client v6 Changes'.\n\nPlease refer to {0} for more details.\n\nError message: {1}".L10N("Client:Main:NotMigratedClientException"),
-                                                  "https://github.com/CnCNet/xna-cncnet-client/blob/122b2de962afc404e203290d0618363d83c4264a/Docs/Migration-INI.md",
+                                                  "https://github.com/CnCNet/xna-cncnet-client/blob/master/Docs/Migration-INI.md",
```

Comment:
don't refer to a non-permanent link

Reply:
> don't refer to a non-permanent link

There needs to be a more readable link that users can type into their browser, or they need to be able to grab the link from the log file.

[ClientCrashLog_2025_05_12_21_25.txt](https://github.com/user-attachments/files/20174470/ClientCrashLog_2025_05_12_21_25.txt)

Reply:
> > don't refer to a non-permanent link
> 
> There needs to be a more readable link that users can type into their browser, or they need to be able to grab the link from the log file.
> 
> [ClientCrashLog_2025_05_12_21_25.txt](https://github.com/user-attachments/files/20174470/ClientCrashLog_2025_05_12_21_25.txt)

In my opinion, readability is the secondary option when compared with a permanent link. Ensuring readability does not mean we can compromise the permanent link, otherwise, additional constraints are added to the code structures in develop/master branch -- we can't rename the doc folder or the md file anymore.

Reply:
Please, check new text and link